src/components/nav_bar/Navbar.test.tsx

```diff
@@ -39,13 +38,17 @@ describe('when isUserLoggedin is false', () => {
             </TestWrapper>
         );
 
-        expect(() =>
-            getByText(/Log in/i, { selector: 'button' })
-        ).not.toThrow();
+        expect(
+            screen.getByRole('button', {
```

Comment:
Yessss!  I didn't tell you where, are much about specifics, but you still got to the best answer here.  Fuck yeah!!